65black/StackOverflow-lite
# #160381538 Search All Questions

## What does this PR do?
Adds logic for the` POST /api/v1/search` endpoint.

Hitting this endpoint with a search field in your request body will result in an array of questions that matches the content of your search field being returned.

### Description of Task to be completed?
Have the following endpoints working:

> POST /api/v1/search

### How should this be manually tested?
After cloning the repo, cd into it and run `npm run start-dev`. Follow these steps:

 - Using Postman, hit the above endpoint.
- In your request body, add a key-value pair like so: `{search: '<search_term>'}` and hit send.
 - You should get an array of all questions that match your search term.
 - If no question matches your search term, then a 404 response will be returned.

### Screenshots
![search](https://user-images.githubusercontent.com/36507972/45398718-25e7ab00-b63d-11e8-842e-17e44c659ec7.JPG)


### What are the relevant pivotal tracker stories?
#160381538
